config/libc/picolibc.in

```diff
@@ -40,31 +40,47 @@ config LIBC_PICOLIBC_IO_C99FMT
 
 config LIBC_PICOLIBC_REGISTER_FINI
     bool
-    prompt "Enable finalization function registration using atexit"
+    prompt "Enable finalization function registration using atexit [unused]"
```

Comment:
Should we remove these in the next ct-ng release?

Reply:
I hesitated to remove them for fear of breaking existing configurations, but if that wouldn't cause any issues, then, yes, we should remove them. They've never been useful.

Reply:
Yeah I think it's probably a good idea to keep them as no-op for now. Lets see if we can collectively remember to drop them after the next release :)